templates/partials/post_card.rs.html

```diff
@@ -0,0 +1,29 @@
+@use template_utils::*;
+@use plume_models::posts::Post;
+
+@(ctx: BaseContext, article: Post)
+
+<div class="card">
+    @if article.cover_id.is_some() {
+        <div class="cover" style="background-image: url('@Html(article.cover_url(ctx.0).unwrap_or_default())')"></div>
+    }
+    <h3><a href="@article.url(ctx.0)">@article.title</a></h3>
+    <main>
+        <p>@article.subtitle</p>
+    </main>
+    <p class="author">
+        @Html(i18n!(ctx.1, "By {0}"; format!(
```

Comment:
there is an xss injection there, because {0} is not properly escaped. Example paylod: display name = `"><script>alert("xss");</script> <div title="`

Reply:
Yes, I guess we could use `.to_html` from Ructe to escape it https://github.com/kaj/ructe/blob/master/src/template_utils.rs#L24

Reply:
then it should be called only on L18, as to not break the link we build

Reply:
Would doing that break anything?

```rust
i18n!(ctx.1, "By {0}"; format!(
    "<a href=\"/@/{}/\">{}</a>",
    article.get_authors(ctx.0)[0].get_fqn(ctx.0).to_html(),
    article.get_authors(ctx.0)[0].name(ctx.0).to_html()
))
```